src/Server/ChatCommandStore.cpp

```diff
@@ -35,7 +37,7 @@ namespace ewn
 		if (player->GetName() != "Lynix")
 			return false;
 
-		*static_cast<volatile int*>(nullptr) = 42;
+		std::terminate();
```

Comment:
std::terminate doesn't crash the server as would do a segmentation fault/division by zero and is handled by the system on both Windows and Linux.

Maybe a collection of std::raise followed by a call to std::terminate would be better?